script/Arcueid/Day 7/07_00_ARC07_1.txt

```diff
@@ -413,6 +413,7 @@ But, what I need to do right now,
 [sha:c24af6ace10b6849951a176d81ba1afbf4a3dcc4]{
 -- Page 28, Offset 16641.
 -- 秋葉の手を払って、脱兎の如く走り出した。
+// to be rechecked, hare off was suggested
```

Comment:
If we want to keep the 兎 then yeah I think hare off is good

Reply:
Hmm, maybe apply run like the wind here since it's idiomatic, and haring off fits the context better in the other use case (it implies a wildness, an uncoördinatedness)